CONTRIBUTING.rst

```diff
@@ -0,0 +1,678 @@
+    Airflow tests. Breeze provides a preconfigured Docker Compose environment
+    where all these services are available and can be used by tests
+    automatically.
+
+-   Breeze environment is almost the same as used in `Travis CI <https://travis-ci.com/>`__. automated builds.
+    So, if the tests run in your Breeze environment, they will most likely work in Travis CI as well.
+
+Limitations:
+
+-   Breeze environment takes significant space in your local Docker cache. There
+    are separate environments for different Python and Airflow versions, and
+    each of the images takes around 3GB in total.
+
+-   Though Airflow Breeze setup is automated, it takes time. The Breeze
+    environment uses pre-built images from DockerHub and it takes time to
+    download and extract those images. Building the environment for a particular
+    Python version takes less than 10 minutes.
+
+-   Breeze environment runs in the background taking precious resources, such as
+    disk space and CPU. You can stop the environment manually after you use it
+    or even use a ``bare`` environment to decrease resource usage.
+
+::
+
+    Breeze CI images are not supposed to be used in production
+    environments. They are optimized for repeatability of tests, maintainability
+    and speed of building rather than production performance. The production images are not yet
+    officially published.
+
+Pylint Checks
+=============
+
+::
+
+    We are in the process of fixing code flagged with pylint checks
+    for the whole Airflow project. This is a huge task so we implemented an
+    incremental approach for the process. Currently most of the code is excluded
+    from pylint checks via scripts/ci/pylint_todo.txt.
+    We have an open JIRA issue AIRFLOW-4364 which has
+    a number of sub-tasks for each of the modules that should be made
+    compatible. Fixing problems identified with pylint is one of straightforward
+    and easy tasks to do (but time-consuming), so if you are a first-time
+    contributor to Airflow, you can choose one of the sub-tasks as your first
+    issue to fix.
+
+
+The process to fix a pylint issue looks as follows:
+
+1.  Remove module/modules from the
+    `scripts/ci/pylint_todo.txt <scripts/ci/pylint_todo.txt>`__.
+
+2.  Run `scripts/ci/ci_pylint.sh <scripts/ci/ci_pylint.sh>`__.
+
+3.  Fix all the issues reported by pylint.
+
+4.  Re-run `scripts/ci/ci_pylint.sh <scripts/ci/ci_pylint.sh>`__.
+
+5.  If you see "success", submit a PR following
+    `Pull Request guidelines <#pull-request-guidelines>`__.
+ 
+
+These are guidelines for fixing errors reported by pylint:
+
+-   Fix the errors rather than disable pylint checks. Often you can easily
+    refactor the code (IntelliJ/PyCharm might be helpful when extracting methods
+    in complex code or moving methods around).
+
+-   If disabling a particular problem, make sure to disable only that error by
+    using the symbolic name of the error as reported by pylint.
```

Comment:
Elena: //disable=exclude it from the scan? Is this a known term?
Jarek: I added an example below. It is a known term for people using pylint. With the example it should be clear :)